sh/install.sh

```diff
@@ -76,6 +79,9 @@ function install_python() {
     elif [ $OPERATING_SYSTEM == "REDHAT" ]; then
         echo "Installing Python using yum"
         yum install python -y >> ${INSTALL_LOG} 2>&1
+    elif [ $OPERATING_SYSTEM == "openSUSE" ]; then
+        echo "Installing Python using zypper"
+        zypper install python >> ${INSTALL_LOG} 2>&1
```

Comment:
If python 2.7 is existing by default in openSUSE - is this some thing that can be skipped? 
Could this cause python 3 being installed by zypper?